src/plugin-dynamic-import-retry/index.ts

```diff
@@ -0,0 +1,92 @@
+import type { HtmlTagDescriptor, Plugin } from 'vite'
+import { simple } from 'acorn-walk'
+import MagicString from 'magic-string'
+import { createFilter } from '@rollup/pluginutils'
+
+import { createCSSPreloadRetry } from './css-preload-retry'
+import { createImportWrapper } from './import-wrapper'
+
+const dynamicImportPrefixRE = /import\s*\(/
```

Comment:
suggestion: I think we need a more explicit match here

Reply:
@adamdehaven I didn’t notice earlier, but this regexp is just used for quickly bailing out before parsing, so it should be fine.